docs/pages/careers/developer-advocate.md

```diff
@@ -1,116 +1,111 @@
-We have underinvested in this area for too long.
-Your mission will be to be the voice of the developers of the community inside the company.
-We are looking for someone that can contribute to the following outcomes:
+We're also making ambitious moves to incorporate our full suite of components into Toolpad, a low-code admin builder tool for assembling full-stack apps faster than ever.
 
-- Create momentum in the React community and drive adoption of the library.
-- Improve the overall developer experience, resulting in better NPS & CSAT scores, e.g., through educational content.
-- Amplify the need of the community so the product direction aims at what people need the most.
+Learn more about MUI's products in this blog post: [An introduction to the MUI ecosystem](https://mui.com/blog/mui-product-comparison/). (This is also the kind of content you might create!)
 
-Both our open-source community and our premium products are growing fast (x2 YoY).
-We need talented people to keep that going!
+## The culture
 
-## About the role
+MUI is a fully remote company with a team that spans the globe.
+The majority of our work is asynchronous, and we rely on written communication to collaborate.
+We're radically transparent: nearly all of our work happens in public.
+Each contributor has the freedom to decide how and when they work, and that work is primarily self-directed: it's your responsibility to define and complete your own tasks in a timely manner.
 
-We are looking for a experienced developer to join us as a Developer Advocate to help the developers get familiar with and use MUI most effectively.
+Learn more about what we value as an organization: [Evolving MUI's core values and behaviors](https://mui.com/blog/2023-mui-values/).
 
-### Why this is interesting
+## The role
 
-Our products empower React developers to build awesome applications faster – we see millions of developers on MUI's docs every year, one million a month.
+MUI's Developer Experience program consists of three key pillars—Education, Documentation, and Community—which all contribute to the company's primary goal: to help developers build beautiful user interfaces quickly and efficiently.
 
-### What you'll do on a day-to-day basis
+As our next Developer Advocate, you will focus your efforts on education and documentation initiatives, though you may also work on projects related to product DX and community relations from time to time as well.
 
-Depending on the day, you'll:
+Depending on your skills and interests, you may choose to pursue one of two paths on the DevEx team: content (creating educational resources) or infrastructure (improving the DX of the product docs and marketing site).
+(We expect to hire specialists for both of these paths eventually, so we're open to hiring for either one right now.)
```

Comment:
This is something @oliviertassinari and I have been talking about, but we can remove this part (or edit it to specify one of these two paths) if we don't want to proceed with this idea